src/app.js

```diff
@@ -32,6 +33,10 @@ async function app (config) {
   const taskProvider = config.tasks.provider.provider
   app.use('/tasks/', await TaskProvider.mount(taskProvider, config.tasks, config))
 
+  // Add a route to handle the `/messages` route
+  const messageProvider = config.messages.provider.provider
+  app.use('/message/', await MessageProvider.mount(messageProvider, config.messages, config))
```

Comment:
I think `/messages/`  would be more consistent with our other resource paths and better convey a multitude of messages (and your comment also references `/messages` as the route name). Do you mind updating the path to `/messages/`?